db/schema.rb

```diff
@@ -55,6 +55,34 @@
     t.index ["affiliate_id"], name: "index_accounts_on_affiliate_id"
   end
 
+  create_table "active_storage_attachments", charset: "utf8mb3", force: :cascade do |t|
```

Comment:
When I ran `rails db:migrate`, these active storage and affiliate changes happened, though it didn't seem there was a specific migration that was run to make these changes. So I couldn't figure out how to prevent them and also migrate the relays change

Reply:
@jossim try running `db:rollback` then check the schema, repeat until the active storage and affiliate changes are removed.  at that point you may have to get a little creative - perhaps deleting or commenting out the active storage migration - then run `db:migrate` again.